examples/spaces-live-cursors/javascript/README.md

```diff
@@ -1,6 +1,23 @@
 # Tracking users' cursors in an application
 
-This folder contains the code for live cursors (Typescript) - a demo of how you can leverage [Ably Spaces](https://github.com/ably/spaces) to track other member's cursors in realtime on a page.
+Label and track the cursors of users in an application.
+
+Track and display the position of users' cursors within an application to provide visual cues of other users' activities and areas of focus in shared workspaces.
+
+Live cursors are utilized in applications such as online whiteboards to enable users to see where others are pointing or what they are drawing. They are also used in slideshow applications to enable users to see which slide each member is interacting with.
+
+Live cursors are implemented using [Ably Spaces](/docs/products/spaces). The Spaces SDK contains a set of purpose-built APIs that abstract away the complexities involved in managing the state of participants in a collaborative application. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.
+
+## Resources
+
+Use the following components to add live cursors into an application:
+
+* [`spaces.get()`](/docs/spaces/space#create) - creates a new or retrieves an existing `space`.
```

Comment:
Another request if you don't mind - discovered this after my initial review. Having hyphens here within the list items confuses the markdown parsing process as it's the syntax for another list.

To help this, and to be more consistent with the react example, can these be changed to colons?

Reply:
Sure. I've updated it.